includes/settings-page.php

```diff
@@ -0,0 +1,83 @@
+<?php
+add_action('admin_menu', 'acf_recaptcha_add_admin_menu');
+add_action('admin_init', 'acf_recaptcha_settings_init');
+
+
+function acf_recaptcha_add_admin_menu()
+{
+    add_options_page(__('ACF reCAPTCHA', 'acf-recaptcha'), __('ACF reCAPTCHA', 'acf-recaptcha'), 'manage_options', 'acf-recaptcha', 'acf_recaptcha_options_page');
+}
+
+
+function acf_recaptcha_settings_init()
+{
+    register_setting('acf_recaptcha_settings', 'acf-recaptcha');
```

Comment:
Likewise, the setting name should be `acf_recaptcha`, I think this is a typo.

Reply:
I took a look at a number of other plugins I have installed. Most of their settings seem to use - rather than _ as separators for page names. I'm not attached to either way, so if this looks wrong to you I'll change it. This is your plugin after all ;)

Reply:
Oh, actually it's because the settings can't save, since you're using`get_option(acf_recaptcha)` and the inputs' names are `acf_recaptcha[site_key]`. You'll need to either fix the incorrect option name references, or change this part.

I'm taking reference from https://gist.github.com/annalinneajohansson/5290405, by the way.

Reply:
I tried it as it was, and nothing was saved to the database until I made this change.